runtime/near-vm-errors/src/lib.rs

```diff
@@ -25,6 +28,12 @@ pub enum VMRunnerError {
 }
 
 /// Permitted errors that cause a function call to fail gracefully.
+///
+/// This error will be included in the merklize state on chain, it must be
+/// deterministically produces on all nodes.
```

Comment:
Let's be a bit more precise here:

The _fact_ that this error occurred (one bit) will be included in the merklized state on chain. Specific details about error are not part of the chain, but they are stored in the database and returned on best-effort basis through RPC.

Reply:
corrected now, thanks for catching this!